src/test/reducer.test.js

```diff
@@ -0,0 +1,74 @@
+const {ADD, DELETE, UPDATE_COUNT, START, END} = require("../constants");
+const {reducer} = require("../reducers/reducer");
+
+test('pass addImages action', () => {
+    const state = {
+        images: ["first", "second"]
+    };
+    const action = {
+        type: ADD,
+        data: ["third", "fourth"]
+    }
+    expect(reducer(state, action)).toStrictEqual({...state, images: [...state.images, ...action.data]});
```

Comment:
```suggestion
    expect(reducer(state, action)).toStrictEqual({ images: ["first", "second", "third", "fourth"] });
```
IMO tests should be simple and avoid unnecessary operations (or one will need to write tests that verify tests =)).